src/tests/PetAdding.spec.ts

```diff
@@ -0,0 +1,77 @@
+import PetList from '../routes/dashboard/pet-list/PetList.svelte'
+import { IdForPet } from '../mocks/data/pets'
+import { format, addMonths } from 'date-fns'
+import '@testing-library/jest-dom'
+import {
+  render,
+  fireEvent,
+  waitFor,
+  getByTestId,
+  getByText,
+  queryByTestId,
+} from '@testing-library/svelte'
+
+describe('As a pet owner I want to add new pet so I can see it among rest of my pets.', () => {
+  const petFormTestId = 'create-pet-form'
+
+  it('GIVEN pet list WHEN pet owner enters the list THEN no form for adding pet is availible', () => {
```

Comment:
I think this could work.

```suggestion
  it(`GIVEN pet list 
        WHEN pet owner enters the list
        THEN no form for adding pet is available`, () => {
```

Reply:
Do you mean multiline?

Reply:
Yes, especially that this is how I've seen most of `BDD` scenarios. GitHub actually processed it a bit strange. Just consider using backticks to format it this way, of course, if you feel it's fine.

Reply:
looks nice!